src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -149,4 +149,15 @@ public void testOuterTypeNumber() throws Exception {
         testByConfig(
                 "sizes/OuterTypeNumber/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/146")
```

Comment:
Why it is ignored?

Reply:
https://github.com/checkstyle/patch-filters/pull/147#issue-456147762